src/EnergyPlus/GroundHeatExchangers.cc

```diff
@@ -1284,8 +1262,8 @@ void GLHEVert::calcShortTimestepGFunctions(EnergyPlusData &state)
             if (cell_index == 0) {
                 // heat flux BC
 
-                auto &thisCell = Cells[cell_index];
-                auto &eastCell = Cells[cell_index + 1];
+                auto const &thisCell = Cells[cell_index];
+                auto const &eastCell = Cells[cell_index + 1];
```

Comment:
[src/EnergyPlus/GroundHeatExchangers.cc:1287]:(style),[constVariable],Variable 'thisCell' can be declared as reference to const
[src/EnergyPlus/GroundHeatExchangers.cc:1288]:(style),[constVariable],Variable 'eastCell' can be declared as reference to const